data/human/deep missions.txt

```diff
@@ -1498,91 +1594,25 @@ mission "Deep: Remnant 1C"
-		accept
-	
-	label 1C1 # The choice of 1C
-	choice
-		`	"I already explored the region beyond the wormhole while you were researching."`
-			goto AC2
-		`	"Sorry, I don't want to work for you anymore."`
-			decline
-	
-	label 2A1 # Only appears in Remnant 2A
-	`When you return, Ivan and the rest of the scientists wait eagerly for you to explain what you discovered.`
-	`	You begin by recounting your experience with the strange space-dwelling creatures you found in the Nenia system. Ivan's eyes go wide when you mention the ease with which they fed on asteroids, and how they seemed to effortlessly escape from the gravity of local gas giants.`
-		goto AC2
-	
-	label AC2 # The end of 2A and 1C
-	branch 2A2 1C2 # Are we in mission 2A or 1C?
-		has "Deep: Remnant 1A: done"
-	
-	label 2A2
-	branch friendly1
-		has "event: ember waste label"
-	`	You then tell them about the strange-looking humans who communicate in song and sign language that attacked you after you wouldn't submit to a blood test.`
-		goto AC3
-				
-	label friendly1
-	`	You then tell them about the Remnant, a group of friendly humans who fled human space during the Alpha Wars and have advanced independently of humanity for centuries, and how they call the region of space the "Ember Waste."`
-		goto AC3
-	
-	label 1C2
-	branch friendly2
-		has "event: ember waste label"
-	`	The scientists look very surprised, and ask that you tell them what you found. You recall your encounter with strange-looking humans who communicate in song and sign language and attacked you after you wouldn't submit to a blood test.`
-		goto AC3
-				
-	label friendly2
-	`	The scientists look very surprised, and ask that you tell them what you found. You recall your encounter with the Remnant, a group of friendly humans who fled human space during the Alpha Wars and have advanced independently of humanity for centuries, and how they call the region of space the "Ember Waste."`
-		goto AC3
-	
-	label AC3
-	`	The group of scientists looks captivated the entire time, almost like kindergarten children listening to a fairy tale. When you finish talking, they immediately chatter among themselves, seeming to be most interested in the human beings which inhabit the region. They seem to agree that they need to learn more about these humans, and more specifically about the technology which they have developed.`
-	`	They eventually come to the conclusion that you should return to this region of space and scan the ships of these humans so that they can learn more about them.`
+conversation "deep: remnant c"
+	`The team of scientists is already waiting in the spaceport when you arrive. They remove the equipment from your ship and immediately being analyzing the results after returning to the lab.`
+	`	"At first glance this series of wormholes seems no different than the one that is used to enter this region of space, but it will require some time for us to go over all the data the sensors collected," Ivan explains to you.`
```

Comment:
```suggestion
	`	"At first glance, this series of wormholes seems no different than the one that is used to enter this region of space, but it will require some time for us to go over all the data the sensors collected," Ivan explains to you.`
```